examples/flutter_gallery/lib/demo/cupertino/cupertino_dialog_demo.dart

```diff
@@ -92,6 +94,108 @@ class _CupertinoDialogDemoState extends State<CupertinoDialogDemo> {
+                    ),
+                    new CupertinoDialogAction(
+                      child: const Text('Oatmeal Cookie'),
+                      onPressed: () { Navigator.pop(context, 'Oatmeal Cookies'); }
+                    ),
+                    new CupertinoDialogAction(
+                      child: const Text('Chocolate Brownie'),
+                      onPressed: () { Navigator.pop(context, 'Chocolate Brownies'); }
+                    ),
+                    new CupertinoDialogAction(
+                      child: const Text('Cancel'),
+                      isDestructiveAction: true,
+                      onPressed: () { Navigator.pop(context, 'Cancel'); }
+                    ),
+                  ]
+                ),
+              );
+            },
+          ),
+          const Padding(padding: const EdgeInsets.all(8.0)),
+          new CupertinoButton(
+            child: const Text('Alert Buttons Only'),
+            color: CupertinoColors.activeBlue,
+            padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
+            onPressed: () {
+              showDemoDialog<String>(
+                context: context,
+                child: new CupertinoAlertDialog(
+                  actions: <Widget>[
+                    new CupertinoDialogAction(
+                      child: const Text('One'),
+                      onPressed: () { Navigator.pop(context, 'One'); }
+                    ),
+                    new CupertinoDialogAction(
+                      child: const Text('Two'),
+                      onPressed: () { Navigator.pop(context, 'Two'); }
+                    ),
+                    new CupertinoDialogAction(
+                      child: const Text('Three'),
+                      onPressed: () { Navigator.pop(context, 'Three'); }
+                    ),
+                    new CupertinoDialogAction(
+                      child: const Text("Four"),
+                      onPressed: () { Navigator.pop(context, "Four"); }
+                    ),
+                    new CupertinoDialogAction(
+                      child: const Text('Five'),
+                      onPressed: () { Navigator.pop(context, 'Five'); }
+                    ),
+                    new CupertinoDialogAction(
+                      child: const Text('Six'),
+                      onPressed: () { Navigator.pop(context, 'Six'); }
+                    ),
+                    new CupertinoDialogAction(
+                      child: const Text('Seven'),
+                      onPressed: () { Navigator.pop(context, 'Seven'); }
+                    ),
+                    new CupertinoDialogAction(
+                      child: const Text('Cancel'),
+                      isDestructiveAction: true,
+                      onPressed: () { Navigator.pop(context, 'Cancel'); }
+                    ),
+                  ]
+                ),
+              );
+            },
+          ),
```

Comment:
for clarity, maybe factor out the desert dialog, and just have both examples call that code. That way they both have interesting dialogs (the numbers aren't super interesting compared to deserts!).

Reply:
:)